packages/Path-View-Core.package/SClassDiagram.class/instance/setSemanticZooming..st

```diff
@@ -0,0 +1,11 @@
+zooming
+setSemanticZooming: aNumber
+
+	| fontSize |
+	self isSemanticZooming 
```

Comment:
can this happen? For me it looks like that this method is only called if semanticZooming is enabled